tf2onnx/optimizer/transpose_optimizer.py

```diff
@@ -822,6 +824,28 @@ def permute_pads(pads):
 
     def _prelu_handler(self, trans, node):
         return self._handle_node_having_branches(trans, node)
+    
+    def _softmax_handler(self, trans, node):
+        trans_rank = get_transpose_rank(trans)
+        perm = trans.get_attr("perm").ints
+
+        if self._g.opset >= 13:
+            # Softmax operates on an arbitrary axis since opset 13
+            axis = node.get_attr_value("axis", -1)
+            new_axis = perm[axis + trans_rank if axis < 0 else axis]
+            if not self._switch_transpose_and_node(node, trans):
```

Comment:
```suggestion
            if not self._simple_through_handler(trans, node):
```

Reply:
I think those are identical

Reply:
Yes, the 2 function is identical but _simple_through_handler seems like a wrapper for such case. Just like we call this method for some simple ops instead of calling _switch_transpose_and_node().

Reply:
_simple_through_handler seems to only be used in _handler_map, the other handler functions always use _switch_transpose_and_node.

Thus I would prefer to leave this as is.